tests/validator/index-test.js

```diff
@@ -0,0 +1,116 @@
+'use strict'
+
+const expect = require('chai').expect
+const validator = require('../../lib/validator')
+
+describe('validator', () => {
+  ;[
```

Comment:
Do you mind giving a short description (via comments or a description string) as to what each of these test cases are testing?